blog-posts/blog4.qmd

```diff
@@ -4,3 +4,18 @@ description: ""
 date: "11/14/22"
 editor: visual
 ---
+
+My name is Josh Yamamoto and I'm finishing up my final year as a Mathematics & Statistics major at Reed College. I have been lucky enough to interact with data science in a wide variety of settings and flavors. Out of that experience grew my primary interest: a deep passion for propagating thoughtfulness and intentionality in all of the ways that data science gets done. While I have a special love for solving very technical problems, I care just as much about how those problems get solved.
```

Comment:
```suggestion
My name is Josh Yamamoto and I'm finishing up my final year as a Mathematics & Statistics major at Reed College. I have been lucky enough to interact with data science in a wide variety of settings and flavors. Out of those experiences grew my primary interest: a deep passion for propagating thoughtfulness and intentionality in all of the ways that data science gets done. While I have a special love for solving very technical problems, I care just as much about how those problems get solved.
```

Reply:
I think that you ought to forefront that you're looking for a job within the first sentence or two. I think it might also be nice to see a glimpse of your professional identity independent of your degree. Another way this could flow:

"My name is Josh Yamamoto. I'm a statistical software developer with an appreciation for interpersonal connection and principled data science workflows. After graduation from Reed College this spring with a degree in Mathematics & Statistics, I'll be on the job market for roles in statistical programming and data science."